pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -67,24 +67,29 @@ func (mtp *ManualTickProvider) NewTicker(time.Duration) *time.Ticker {
 	}
 }
 
+// SetStableAndPanicConcurrency sets the stable and panic concurrencies.
+func (mc *MetricClient) SetStableAndPanicConcurrency(s, p float64) {
```

Comment:
should we use same approach for setting `StableRPS` and `PanicRPS` for consistency? (e.g. on L181 of the test)

Reply:
There was just a single usage, seemed like overkill :)

Reply:
hm ok, seems nice to be consistent anyway but hardly important 🤷

Reply:
yeah, net line difference would've been negative :)

Reply:
well, positive :) but that's a negative feature :)

Reply:
:)